scripts/get

```diff
@@ -130,7 +130,12 @@ downloadFile() {
 # installs it.
 installFile() {
   HELM_TMP="/tmp/$PROJECT_NAME"
-  local sum=$(openssl sha -sha256 ${HELM_TMP_FILE} | awk '{print $2}')
+  local shac="sha1"
+  local sha1exist=$(openssl sha -noop 2>&1 | grep sha1 | wc -l)
```

Comment:
- [ ] If _grep_ returns 1, no lines matched.  
  You can check the exit status of _grep_, rather than counting the lines that it printed.  
```shell
if ! openssl list-message-digest-commands | grep --silent '^sha1$'; then
  shac='sha'
fi
```

Reply:
Nice. Thanks!